core/src/main/scala/com/nvidia/spark/rapids/tool/qualification/Qualification.scala

```diff
@@ -85,6 +85,9 @@ class Qualification(outputDir: String, numRows: Int, hadoopConf: Configuration,
     }
     qWriter.writeExecReport(allAppsSum, order)
     qWriter.writeStageReport(allAppsSum, order)
+    if (allAppsSum.exists(x => x.mlFunctions.nonEmpty)) {
+      qWriter.mlFuncsReport(allAppsSum, order)
```

Comment:
nit would be nice to have this be writeMlFuncsREports to be consistent with others